gen_core_pools.py

```diff
@@ -0,0 +1,77 @@
+import json
+
+import requests
+
+from gen_pools_and_gauges import get_subgraph_url
+
+
+def get_stable_pools_with_rate_provider():
+    """
+    for every chain, query the official balancer subgraph and retrieve pools that:
+    - have a rate provider different from address(0)
+    - have a liquidity greater than $250k
+    - have a yield fee > 0
```

Comment:
Still need to add 
- Is added to the gauge controller and is not killed as per comments  in the main discussion.

Reply:
see [098ea82](https://github.com/BalancerMaxis/bal_addresses/pull/177/commits/098ea82dc88b9e6c244cc508b2ce7ae9cc9872ce)